tests/unit/views/ont/ONTPoolIndex.spec.js

```diff
@@ -55,49 +67,36 @@ describe('OntPoolIndex', () => {
   })
 
   describe('Printing labels', () => {
-    beforeEach(() => {
-      pools = wrapper.vm
-      pools.selected = [
-        { id: 1, barcode: 'TRAC-2-1', source_identifier: 'SQSC-1' },
-        { id: 2, barcode: 'TRAC-2-2', source_identifier: 'SQSC-2' },
-        { id: 3, barcode: 'TRAC-2-3', source_identifier: 'SQSC-2' },
-      ]
-    })
-
-    describe('#createLabels', () => {
-      it('will have the correct number of labels', () => {
-        expect(pools.createLabels().length).toEqual(3)
-      })
-
-      it('will have the correct text for each label', () => {
-        const label = pools.createLabels()[0]
-        expect(label.barcode).toEqual('TRAC-2-1')
-        expect(label.first_line).toEqual('Ont - Pool')
-        expect(/\d{2}-\w{3}-\d{2}/g.test(label.second_line)).toBeTruthy()
-        expect(label.third_line).toEqual('TRAC-2-1')
-        expect(label.fourth_line).toEqual('SQSC-1')
-        expect(label.round_label_bottom_line).toEqual('1')
-        expect(label.round_label_lower_line).toEqual('TRAC-2')
-        expect(label.label_name).toEqual('main_label')
+    describe('#onPrintAction', () => {
+      beforeEach(() => {
+        pools = wrapper.vm
+        pools.selected = [
+          { id: 1, barcode: 'TRAC-1', source_identifier: 'SQSC-1' },
+          { id: 2, barcode: 'TRAC-2', source_identifier: 'SQSC-2' },
+          { id: 3, barcode: 'TRAC-2', source_identifier: 'SQSC-2' },
```

Comment:
just checking it's correct that 2 and 3 are identical, as they had different barcodes on old version

Reply:
Ah yes good spot, that was unintentional - fixed in [0f92f73](https://github.com/sanger/traction-ui/pull/2563/commits/0f92f73626e8dfdeb9795da0b60ac2ba1251590e)